server/datastore/mysql/policies.go

```diff
@@ -421,8 +456,17 @@ func (ds *Datastore) NewTeamPolicy(ctx context.Context, teamID uint, authorID *u
 	return policyDB(ctx, ds.writer, uint(lastIdInt64), &teamID)
 }
 
-func (ds *Datastore) ListTeamPolicies(ctx context.Context, teamID uint) ([]*fleet.Policy, error) {
-	return listPoliciesDB(ctx, ds.reader, &teamID)
+func (ds *Datastore) ListTeamPolicies(ctx context.Context, teamID uint) (teamPolicies, inheritedPolicies []*fleet.Policy, err error) {
+	teamPolicies, err = listPoliciesDB(ctx, ds.reader, &teamID, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	// get inherited (global) policies with counts of hosts for that team
+	inheritedPolicies, err = listPoliciesDB(ctx, ds.reader, nil, &teamID)
+	if err != nil {
+		return nil, nil, err
+	}
+	return teamPolicies, inheritedPolicies, err
```

Comment:
just to double check, if a host doesn't belong to a team, both `teamPolicies` and `inheritedPolicies` will be the same?

Reply:
Not sure I understand your question - this lists the policies of a team, it's not related to a host. In all cases, `teamPolicies` will contain the policies created for that specific team, while `inheritedPolicies` are the global policies (those not associated with a team). 

Only the passing/failing counts are related to hosts, was your question related to those counts?

Reply:
sorry about that, I was confused and it was a poorly written question, your comment above answers my question really:

```
// The pass/fail host counts are the totals
// regardless of hosts' team if countsForTeamID is nil, or the totals just for
// hosts that belong to the provided countsForTeamID if it is not nil.
```